scene/animation/animation_blend_space_1d.cpp

```diff
@@ -303,15 +303,15 @@ double AnimationNodeBlendSpace1D::process(double p_time, bool p_seek, bool p_is_
 				if (point_lower == -1) {
 					point_lower = i;
 					pos_lower = pos;
-				} else if ((blend_pos - pos) < (blend_pos - pos_lower)) {
+				} else if (pos < pos_lower) {
```

Comment:
```suggestion
				} else if (pos > pos_lower) {
```
The transpose in the inequality flips the direction.

Reply:
right, thanks.